src/GovUk.Education.ExploreEducationStatistics.Admin/ViewModels/AddUserReleaseRoleViewModel.cs

```diff
@@ -5,7 +5,7 @@
 
 namespace GovUk.Education.ExploreEducationStatistics.Admin.ViewModels
 {
-    public class AddReleaseRoleViewModel
+    public class AddUserReleaseRoleViewModel
```

Comment:
I've renamed it to `UserReleaseRoleAddViewModel`.

I added the `User` to distinguish between `ReleaseRole`s like `Approver` and an object that contains a release ID and a release role. `UserReleaseRole` and `UserPublicationRole` are commonly used for this type of object elsewhere in the codebase. I think `User` does make sense because these objects are attached to specific users. It's possible there is a better way to name it - something like `ReleaseRolePermission`? - but with so much of the codebase already using `UserReleaseRole` already, my preference would be to stick to that.